rfc6962-bis.xml

```diff
@@ -572,6 +581,15 @@ TBSCertificate from it, redacting the domain name labels indicated by the
 redacted labels extension, and deleting the SCT list extension and redacted
 labels extension.
         </t>
+        <t>
+          <spanx style="verb">id</spanx> identifies a single extension from the IANA registry in <xref target="extension_ids"/>.
```

Comment:
Actually, inspired by reviewing RFC5246 Section 7.4.1.4 again just now, I think we should do this...

    enum {
        reserved(65535)
    } CtExtensionType;

    struct {
        CtExtensionType extension_type;
        opaque extension_data<0..2^16-1>;
    } CtExtension;

    CtExtension extensions<0..2^16-1>;

OK?